src/cloudflare/internal/d1-api.ts

```diff
@@ -208,13 +210,9 @@ class D1DatabaseSession {
       init.headers = h;
     }
     return this.fetcher.fetch(input, init).then((resp) => {
-      const newCommitToken = resp.headers.get(
-        D1_SESSION_COMMIT_TOKEN_HTTP_HEADER
-      );
-      // TODO(soon): Add validation of the received commit token, in case we sent a commit token,
```

Comment:
Just noting that it's OK to lose this TODO because the internal DO methods for `waitForBookmark` should validate the token for D1.